tests/ctst/features/iam-policies/IAMUser.feature

```diff
@@ -0,0 +1,82 @@
+Feature: IAM Policies for IAM Users
+  This feature allows you to create and attach IAM policies for IAM users.
+  IAM users should have the permissions to perform the actions that they are granted in their IAM policies.
+
+  @2.6.0
+  @PreMerge
+  @IAM-Policies-IAMUsers
+  Scenario Outline: User is not authorized to perform the actions without IAM policy
+    Given an existing bucket "<bucketName>" "without" versioning
+    And we "<haveObject>" an existing object "<objectName>" "without" version-Id ""
```

Comment:
Not sure to understand a sentence like "And we havenot an existing object ObjectName without version-Id"

how about this:
```suggestion
    And a object that "<objectExists>" "<without>" version-Id
```

* The `"<objectExists>"` can be either `exists` or `does not exists`
* The object name is not needed unless we want to test a specific object name, it can be generated by the test
* Same for the version id

Reply:
Will update in common PR, starting here ed5d2bd2e26929a6a1decb7086b79ade19e1e988